test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -31,7 +31,8 @@ import (
 
 const (
 	qatPluginKustomizationYaml = "deployments/qat_plugin/overlays/e2e/kustomization.yaml"
-	opensslTestYaml            = "demo/openssl-qat-engine-cpa-sample-pod.yaml"
+	opensslTestCyYaml          = "demo/openssl-qat-engine-cpa-sample-cy-pod.yaml"
+	opensslTestDcYaml          = "demo/openssl-qat-engine-cpa-sample-dc-pod.yaml"
```

Comment:
does not make much sense to duplicate these files if only one line is different. could you perhaps follow something like https://github.com/intel/intel-device-plugins-for-kubernetes/blob/0bf4cd011247962dc833ea0690a2895a5b26be56/test/e2e/sgx/sgx.go#L94-L111 and make test node specific config for the resource

Reply:
fixed!

Reply:
Looks good but why not share the same with the crypto test? The test return ok but complains about missing compress instances. Is that expected?

Reply:
Umm, what would "share" mean?
And,, I can see the complains about missing compress instances, which I cannot see in my SPR environment.
It's not expected... ;(

Reply:
drop `opensslTestYamlPath` and use the same `podSpec` "template" for both 'cpa_sample` tests

Reply:
Ah, ok understood.! :)
Thanks for the explanation.!

Reply:
the problem about 'missing compress instances' was because of no 'Requests'. There was only 'Limits', which is same as the openssl yaml and worked fine in a real HW.